server/block/break_info.go

```diff
@@ -236,6 +237,99 @@ func silkTouchOnlyDrop(it world.Item) func(t item.Tool, enchantments []item.Ench
 	}
 }
 
+// fortuneLevel returns the level of the fortune enchantment in enchantments, or 0 if it isn't present.
+func fortuneLevel(enchantments []item.Enchantment) int {
+	index := slices.IndexFunc(enchantments, func(i item.Enchantment) bool {
+		return i.Type() == enchantment.Fortune
+	})
+	if index == -1 {
+		return 0
+	}
+	return enchantments[index].Level()
+}
+
+// hasFortune checks if an item has the fortune enchantment.
+func hasFortune(enchantments []item.Enchantment) bool {
+	return slices.IndexFunc(enchantments, func(i item.Enchantment) bool {
+		return i.Type() == enchantment.Fortune
+	}) != -1
+}
```

Comment:
Why not just `fortuneLevel(enchantments) != 0`? This fn only called twice and both cases have a `fortuneLevel` call anyway